src/System.IO.FileSystem/tests/FileStream/ctor_str_fm_fa_fs_buffer_fo.cs

```diff
@@ -29,19 +29,19 @@ public void InvalidFileOptionsThrow()
         [ConditionalTheory(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
         [InlineData(FileOptions.None)]
         [InlineData(FileOptions.DeleteOnClose)]
-        [InlineData(FileOptions.Encrypted)]
+        //[InlineData(FileOptions.Encrypted)] may not be allowed for some users
         [InlineData(FileOptions.RandomAccess)]
         [InlineData(FileOptions.SequentialScan)]
         [InlineData(FileOptions.WriteThrough)]
         [InlineData((FileOptions)0x20000000)] // FILE_FLAG_NO_BUFFERING on Windows
         [InlineData(FileOptions.Asynchronous)]
         [InlineData(FileOptions.Asynchronous | FileOptions.DeleteOnClose)]
-        [InlineData(FileOptions.Asynchronous | FileOptions.Encrypted)]
+        //[InlineData(FileOptions.Asynchronous | FileOptions.Encrypted)] may not be allowed for some users
```

Comment:
I suggest just deleting this line.  The notion that Encrypted may not work is already covered by the line that's for just Encrypted.